compiler/rustc_const_eval/src/check_consts/check.rs

```diff
@@ -694,6 +695,88 @@ impl<'tcx> Visitor<'tcx> for Checker<'_, 'tcx> {
                     }
                 };
 
+                let check_stability = |this: &mut Self, def_id| {
```

Comment:
```suggestion
                // Check the const stability of the given item (fn or trait).
                let check_item_stability = |this: &mut Self, def_id| {
```
Could this be a proper method or would that duplicate too many arguments?

Reply:
Then maybe this should be `check_callee_stability` as well.

Reply:
Done and extracted to a proper method